game-core/src/main/java/games/strategy/engine/gamePlayer/DefaultPlayerBridge.java

```diff
@@ -64,33 +68,30 @@ public IRemote getRemoteDelegate() {
       game.getData().acquireReadLock();
       try {
         final IDelegate delegate = game.getData().getDelegateList().getDelegate(currentDelegate);
-        if (delegate == null) {
-          final String errorMessage = "IDelegate in DefaultPlayerBridge.getRemote() cannot be null. CurrentStep: "
-              + stepName + ", and CurrentDelegate: " + currentDelegate;
-          // for some reason, client isn't getting or seeing the errors, so make sure we print it to err
-          // too
```

Comment:
Comments like this scare me.

Reply:
ya, I've been looking at these for some time:
- note they are mass copy pasted
- before our streams/logging was complex, the two sources we previously logged to now both go to the same place, it's fully redundant to do the double logging now.